app/controllers/application_controller.rb

```diff
@@ -37,6 +42,10 @@ def log_safe_current_user(reload: false)
     @log_safe_current_user
   end
 
+  def development_mode_auth?
+    Settings.key?(:authorised_users)
```

Comment:
should we double lock it...
ie `rails_env != "producation" `

Reply:
Ya, that's in `development_mode_password`, but it's probably worth double checking here too. Although I'd probably still use `Rails.env.development?`.

Reply:
all env are playground apart from `production` as long as she don't shot our foot, I'm fine